tests/cases/aff-alternatives.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../jats.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../support.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error line="7" message="<aff-alternatives> is ignored." node="/aff-alternatives"?>
+<?expected-error line="8" message="<aff> is ignored." node="//aff"?>
```

Comment:
Ideally I'd like to ignore children of sub-elements, but need to be careful with mixed content (they're not ignored in that case).

Reply:
Could separate block and inline elements, and only check children of the latter.